test/lib/reporter_test.rb

```diff
@@ -94,9 +118,41 @@ def decision_question(description)
         [dummy-check-fail]
         [dummy-check-fail2]
 
-      Resolve the failed steps and rerun
-      the command. In case the failures are false positives,
-      use --whitelist=\"dummy-check-fail,dummy-check-fail2\"
+      Resolve the failed steps and rerun the command.
+
+      If the situation persists and, you are unclear what to do next,
+      contact Red Hat Technical Support.
+
+      In case the failures are false positives, use
+      --whitelist=\"dummy-check-fail,dummy-check-fail2\"
+      MESSAGE
+    end
+
+    it 'informs the user about failures of the last scenario for upstream' do
+      assume_feature_absent(:satellite)
+      assume_feature_absent(:capsule)
+      fake_instance_feature
+      run_scenario(fail_multiple_scenario)
+      reporter.after_scenario_finishes(fail_multiple_scenario)
+      assert_equal <<-MESSAGE.strip_heredoc.strip, captured_out(false).strip
+      Check that ends up with fail:                                         [FAIL]
+      this check is always causing failure
+      --------------------------------------------------------------------------------
+      Check that ends up with fail:                                         [FAIL]
+      this check is always causing failure
+      --------------------------------------------------------------------------------
+      Check that ends up with success:                                      [OK]
+      --------------------------------------------------------------------------------
+      Scenario [Scenarios::Dummy::FailMultiple] failed.
+
+      The following steps ended up in failing state:
+
+        [dummy-check-fail]
+        [dummy-check-fail2]
+
+      Resolve the failed steps and rerun the command.
+      In case the failures are false positives, use
+      --whitelist=\"dummy-check-fail,dummy-check-fail2\"
```

Comment:
```suggestion
      --whitelist="dummy-check-fail,dummy-check-fail2"
```

no need to escape those quotes -- makes it easier to read ;)